model-optimizer/telemetry/utils/isip.py

```diff
@@ -25,7 +26,7 @@ def isip_consent_base_dir():
     if platform == 'Windows':
         dir_to_check = '$LOCALAPPDATA'
     elif platform in ['Linux', 'Darwin']:
-        dir_to_check = '$HOME'
+        dir_to_check = Path.home()
```

Comment:
Add a comment here why we get home directory that way

Reply:
Anyway. This PR should not be merged. Instead, the changes should be merged here: https://github.com/openvinotoolkit/telemetry

Reply:
Ok, thank you